app/models/katalyst/content/item.rb

```diff
@@ -33,6 +35,19 @@ def to_plain_text
         heading if show_heading? && visible?
       end
 
+      def show_heading?
+        heading_style != "none"
```

Comment:
If you define an enum for `heading_style` then you'll have access to enum predicates. These are safer than checking the values directly.

E.g.

```
enum heading_style: config.heading_styles, prefix: :heading

alias_method :show_heading?, :heading_none?
```